.github/CODEOWNERS

```diff
@@ -0,0 +1,31 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+* @pierre-sassoulas @DanielNoord @cdce8p @jacobtylerwalls @hippo91
+
+# Order is important. The last matching pattern has the most precedence.
+
+# @DudeNr33
+pylint/pyreverse/*    @DudeNr33
+tests/pyreverse/*    @DudeNr33
+
+# @pierre-sassoulas
+pylint/messages/*    @pierre-sassoulas
+
+# typing
+pylint/typing.py     @DanielNoord @cdce8p
```

Comment:
Instead of ordering by owner, it's better to do by category (core files, checkers, extensions). If a specific checker has tests associated with it (mostly extensions) they should also be grouped together.